codex/blocktype.nim

```diff
@@ -31,7 +31,7 @@ import ./codextypes
 export errors, logutils, units, codextypes
 
 type
-  Block* = object of RootObj
+  Block* = ref object of RootObj
```

Comment:
Why are we changing this to `ref`?

Reply:
Cause otherwise it crashes with a SIGSEV during testing. :-/ I can try to debug it, but that's probably not a quick one.